// @uitk/themes/uhc-solas/js/_variables.js
/**
 * Do not edit directly
 * Generated on Wed, 27 Jul 2022 10:16:53 GMT
 */

export const AliasBorderWidthBase = "1px";
export const AliasBorderWidthMedium = "2px";
export const AliasBorderWidthLarge = "4px";
export const AliasBorderWidthPanels = "1px";
export const AliasBorderWidthButtons = "1px";
export const AliasBorderWidthFormElementBase = "1px";
export const AliasBorderWidthFormElementFocus = "3px";
export const AliasBorderWidthFocus = "5px";
export const AliasBorderRadiusButtons = "1.5rem";
export const AliasBorderRadiusBase = "0.25rem";
export const AliasBorderRadiusFormElement = "0.25rem";
export const AliasBorderRadiusSmall = "0.125rem";
export const AliasBorderRadiusLarge = "0.5rem";
export const AliasBoxshadowFormBase = "0 0 0 1px #737373";
export const AliasBoxshadowFormHover = "0 0 0 1px #196ECF";
export const AliasBoxshadowFormFocus = "0 0 0 3px #196ECF";
export const AliasBoxshadowFormError = "0 0 0 1px #E91B18";
export const AliasBoxshadowFormDisabled = "0 0 0 1px #DADBDC";
export const AliasBoxshadowFormCheckedFocus = "0 0 0 2px #ffffff, 0 0 0 5px #002677";
export const AliasBoxshadowFormCheckedErrorFocus = "0 0 0 2px #F9F1ED, 0 0 0 5px #002677";
export const AliasBoxshadowFormInputFocusError = "0 0 0 3px #E91B18";
export const AliasBoxshadowButtonFocusDefault = "0 0 0 2px #ffffff, 0 0 0 5px #001E5E";
export const AliasBoxshadowButtonFocusAlternative = "0 0 0 2px #ffffff, 0 0 0 5px #1662B8";
export const AliasBoxshadowButtonFocusInverse = "0 0 0 2px #333333, 0 0 0 5px #f4f4f4";
export const AliasBoxshadowButtonFocusDisabled = "0 0 0 2px #f4f4f4, 0 0 0 5px #DADBDC";
export const AliasBoxshadowButtonPressDefault = "0 0 0 2px #ffffff, 0 0 0 5px #001644";
export const AliasBoxshadowButtonPressAlternative = "0 0 0 2px #ffffff, 0 0 0 5px #1456A1";
export const AliasBoxshadowButtonPressInverse = "0 0 0 2px #333333, 0 0 0 5px #DADBDC";
export const AliasBoxshadowButtonGhostDefault = "0 0 0 1px #002677";
export const AliasBoxshadowButtonGhostAlternative = "0 0 0 1px #196ECF";
export const AliasBoxshadowButtonGhostInverse = "0 0 0 1px #ffffff";
export const AliasBoxshadowButtonDisabled = "0 0 0 1px #DADBDC";
export const AliasBoxshadowButtonGroupedDefaultStatic = "0 0 0 5px #ffffff";
export const AliasBoxshadowButtonGroupedDefaultFocus = "inset 0 0 0 3px #002677, inset 0 0 0 6px #ffffff";
export const AliasBoxshadowButtonGroupedDefaultSelected = "0 0 0 3px #001E5E";
export const AliasBoxshadowLinkBase = "0 0 0 3px #196ECF";
export const AliasBoxshadowLinkInverse = "0 0 0 3px #E5F8FB";
export const AliasBoxshadowLinkVisited = "0 0 0 3px #E5F8FB";
export const AliasBoxshadowFocus = "0 0 0 2px #ffffff, 0 0 0 3px #196ECF";
export const AliasBoxshadowTableFocusHeader = "0 0 0 1px #ffffff, inset 0 0 0 2px #196ECF ";
export const AliasBoxshadowTableFocusRow = "0 0 0 1px #ffffff, inset 0 0 0 2px #196ECF ";
export const AliasBoxshadowTableExpandButtonDefault = "0 0 0 1px #5a5a5a";
export const AliasBoxshadowTableExpandButtonFocus = "0 0 0 2px #fafafa, 0 0 0 5px #196ECF";
export const AliasBoxshadowTableExpandButtonPress = "0 0 0 2px #fafafa, 0 0 0 5px #196ECF";
export const AliasBoxshadowNavigationGlobalButtonFocus = "0 0 0 3px #002677";
export const AliasBoxshadowNavigationGlobalButtonPress = "0 0 0 2px #ffffff, 0 0 0 5px #001644";
export const AliasBoxshadowNavigationGlobalButtonStaticFocus = "0 0 0 2px #ffffff, 0 0 0 5px #196ECF";
export const AliasBoxshadowNavigationGlobalButtonStaticPress = "0 0 0 2px #ffffff, 0 0 0 5px #196ECF";
export const AliasBoxshadowTabUnselected = "0 4px 0 0 #F4F4F4";
export const AliasBoxshadowTabSelected = "0 4px 0 0 #002677";
export const AliasBoxshadowPanelsBase = "0 0 0 1px #DADBDC";
export const AliasBoxshadowPanelsAccessible = "0 0 0 1px #737373";
export const AliasBoxshadowPanelsError = "0 0 0 1px #E91B18";
export const AliasPaletteBrand10 = "#E5F8FB";
export const AliasPaletteBrand70 = "#002677";
export const AliasPaletteBrand100 = "#002677";
export const AliasPaletteBrand110 = "#001E5E";
export const AliasPaletteBrand120 = "#001644";
export const AliasPalettePrimary10 = "#E5F8FB";
export const AliasPalettePrimary70 = "#196ECF";
export const AliasPalettePrimary100 = "#002677";
export const AliasPalettePrimary110 = "#001E5E";
export const AliasPalettePrimary120 = "#001644";
export const AliasPaletteSecondary10 = "#E5F8FB";
export const AliasPaletteSecondary70 = "#196ECF";
export const AliasPaletteSecondary100 = "#196ECF";
export const AliasPaletteSecondary110 = "#1662B8";
export const AliasPaletteSecondary120 = "#1456A1";
export const AliasPaletteNeutral0 = "#ffffff";
export const AliasPaletteNeutral10 = "#f4f4f4";
export const AliasPaletteNeutral20 = "#fafafa";
export const AliasPaletteNeutral30 = "#F4F4F4";
export const AliasPaletteNeutral40 = "#DADBDC";
export const AliasPaletteNeutral50 = "#8D8D8D";
export const AliasPaletteNeutral60 = "#737373";
export const AliasPaletteNeutral70 = "#6f6f6f";
export const AliasPaletteNeutral80 = "#6F6F6F";
export const AliasPaletteNeutral90 = "#5a5a5a";
export const AliasPaletteNeutral100 = "#333333";
export const AliasPaletteSuccess10 = "#ECFDEC";
export const AliasPaletteSuccess70 = "#ECFDEC";
export const AliasPaletteSuccess100 = "#0D880B";
export const AliasPaletteSuccess120 = "#0D880B";
export const AliasPaletteSuccessBorder = "#0D880B";
export const AliasPaletteSuccessContent = "#ffffff";
export const AliasPaletteSuccessContrast = "#333333";
export const AliasPaletteError10 = "#F9F1ED";
export const AliasPaletteError70 = "#F9F1ED";
export const AliasPaletteError100 = "#E91B18";
export const AliasPaletteError120 = "#E91B18";
export const AliasPaletteErrorBorder = "#E91B18";
export const AliasPaletteErrorContent = "#ffffff";
export const AliasPaletteErrorContrast = "#333333";
export const AliasPaletteInfo10 = "#E5F8FB";
export const AliasPaletteInfo70 = "#E5F8FB";
export const AliasPaletteInfo100 = "#196ECF";
export const AliasPaletteInfo120 = "#196ECF";
export const AliasPaletteInfoBorder = "#196ECF";
export const AliasPaletteInfoContent = "#ffffff";
export const AliasPaletteInfoContrast = "#333333";
export const AliasPaletteWarning10 = "#F9F1ED";
export const AliasPaletteWarning70 = "#F9F1ED";
export const AliasPaletteWarning100 = "#D14600";
export const AliasPaletteWarning120 = "#D14600";
export const AliasPaletteWarningBorder = "#D14600";
export const AliasPaletteWarningContent = "#ffffff";
export const AliasPaletteWarningContrast = "#333333";
export const AliasPaletteLink10 = "#E5F8FB";
export const AliasPaletteLink70 = "#E5F8FB";
export const AliasPaletteLink100 = "#196ECF";
export const AliasPaletteLink120 = "#002677";
export const AliasPaletteLight = "#ffffff";
export const AliasPaletteDark = "#333333";
export const AliasPaletteFocus = "#196ECF";
export const AliasPaletteActive = "#196ECF";
export const AliasPaletteSurfaceHover = "#002677";
export const AliasPaletteSurfaceTextHover = "#ffffff";
export const AliasPaletteSurfaceBorderHover = "#ffffff";
export const AliasPaletteDisabledBackground = "#f4f4f4";
export const AliasPaletteDisabledBorder = "#DADBDC";
export const AliasPaletteDisabledText = "#8d8d8d";
export const AliasPaletteShadowBase = "51, 51, 51";
export const AliasPaletteShadowBrand = "0, 38, 119";
export const AliasPaletteShadowAccessible = "115, 115, 115";
export const AliasFontSizeHero = "5rem";
export const AliasFontSizeHeadingH1 = "2.75rem";
export const AliasFontSizeHeadingH2 = "2rem";
export const AliasFontSizeHeadingH3 = "1.5rem";
export const AliasFontSizeHeadingH4 = "1.25rem";
export const AliasFontSizeHeadingH5 = "1.125rem";
export const AliasFontSizeHeadingH6 = "1rem";
export const AliasFontSizeBodyTiny = "0.75rem";
export const AliasFontSizeBodySmall = "0.875rem";
export const AliasFontSizeBodyBase = "1rem";
export const AliasFontSizeBodyLarge = "1.125rem";
export const AliasFontSizeBodyLead = "1.5rem";
export const AliasFontSizeFormLabel = "1rem";
export const AliasFontSizeFormHelp = "0.75rem";
export const AliasFontSizeFormError = "0.75rem";
export const AliasFontSizeTableHeader = "1rem";
export const AliasFontSizeTableRow = "1rem";
export const AliasFontSizeNavigationGlobal = "0.75rem";
export const AliasFontSizeNavigationPrimary = "1rem";
export const AliasFontWeightLight = "400";
export const AliasFontWeightRegular = "400";
export const AliasFontWeightMedium = "600";
export const AliasFontWeightBold = "700";
export const AliasFontWeightHeadingH1 = "600";
export const AliasFontWeightHeadingH2 = "600";
export const AliasFontWeightHeadingH3 = "600";
export const AliasFontWeightHeadingH4 = "700";
export const AliasFontWeightHeadingH5 = "700";
export const AliasFontWeightHeadingH6 = "700";
export const AliasFontWeightFormLabels = "700";
export const AliasFontWeightFormHelper = "400";
export const AliasFontWeightFormError = "400";
export const AliasFontFamilyHeadingH1 = "UHC Serif Headline Semibold";
export const AliasFontFamilyHeadingH2 = "UHC Serif Headline Semibold";
export const AliasFontFamilyHeadingH3 = "UHC Serif Headline Semibold";
export const AliasFontFamilyHeadingH4 = "UHC Sans";
export const AliasFontFamilyHeadingH5 = "UHC Sans";
export const AliasFontFamilyHeadingH6 = "UHC Sans";
export const AliasFontFamilyBrand = "UHC Serif";
export const AliasFontFamilyBase = "UHC Sans";
export const AliasFontFamilyData = "UHC Sans";
export const AliasFontFamilyMonotype = "UHC Sans";
export const AliasFontFamilySecondary = "UHC Sans";
export const AliasFontFamilyLabels = "UHC Sans";
export const AliasFontColorHeadingH1 = "#002677";
export const AliasFontColorHeadingH2 = "#002677";
export const AliasFontColorHeadingH3 = "#002677";
export const AliasFontColorHeadingH4 = "#196ECF";
export const AliasFontColorHeadingH5 = "#002677";
export const AliasFontColorHeadingH6 = "#002677";
export const AliasFontColorSubheading = "#196ECF";
export const AliasFontColorHero = "#002677";
export const AliasFontColorBase = "#333333";
export const AliasFontColorInverse = "#ffffff";
export const AliasFontColorSurfaceHover = "#ffffff";
export const AliasFontLetterspacingBase = "0.3rem";
export const AliasFontLetterspacingLoose = "1rem";
export const AliasFontLetterspacingTight = "-0.3rem";
export const AliasFontLineheightBase = "1.5rem";
export const AliasFontLineheightXs = "1rem";
export const AliasFontLineheightS = "1.125rem";
export const AliasFontLineheightM = "1.25rem";
export const AliasFontLineheightL = "1.75rem";
export const AliasFontLineheightXl = "2rem";
export const AliasLogoWidthFooter = "10rem";
export const AliasOpacityLowest = "0.15";
export const AliasOpacityLow = "0.25";
export const AliasOpacityMedium = "0.30";
export const AliasOpacityHigh = "0.50";
export const AliasOpacityHighest = "0.70";
export const AliasSpacingXxs = "0.25rem";
export const AliasSpacingXs = "0.5rem";
export const AliasSpacingSm = "0.75rem";
export const AliasSpacingBase = "1rem";
export const AliasSpacingM = "1.5rem";
export const AliasSpacingLg = "2rem";
export const AliasSpacingXl = "2.5rem";
export const AliasSpacingXxl = "3.5rem";
export const AnimationDurationAnimateSlow = "400ms";
export const AnimationDurationAnimateMedium = "200ms";
export const AnimationDurationAnimateFast = "70ms";
export const BorderWidthBase = "1px";
export const BorderWidthMedium = "2px";
export const BorderWidthLarge = "4px";
export const BorderWidthPanels = "1px";
export const BorderWidthButtons = "1px";
export const BorderWidthFormElementBase = "1px";
export const BorderWidthFormElementFocus = "3px";
export const BorderWidthFocus = "5px";
export const BorderRadiusButtons = "1.5rem";
export const BorderRadiusBase = "0.25rem";
export const BorderRadiusFormElement = "0.25rem";
export const BorderRadiusSmall = "0.125rem";
export const BorderRadiusLarge = "0.5rem";
export const BoxshadowFormBase = "0 0 0 1px #737373";
export const BoxshadowFormHover = "0 0 0 1px #196ECF";
export const BoxshadowFormFocus = "0 0 0 3px #196ECF";
export const BoxshadowFormError = "0 0 0 1px #E91B18";
export const BoxshadowFormDisabled = "0 0 0 1px #DADBDC";
export const BoxshadowFormCheckedFocus = "0 0 0 2px #ffffff, 0 0 0 5px #002677";
export const BoxshadowFormCheckedErrorFocus = "0 0 0 2px #F9F1ED, 0 0 0 5px #002677";
export const BoxshadowFormInputFocusError = "0 0 0 3px #E91B18";
export const BoxshadowButtonFocusDefault = "0 0 0 2px #ffffff, 0 0 0 5px #001E5E";
export const BoxshadowButtonFocusAlternative = "0 0 0 2px #ffffff, 0 0 0 5px #1662B8";
export const BoxshadowButtonFocusInverse = "0 0 0 2px #333333, 0 0 0 5px #f4f4f4";
export const BoxshadowButtonFocusDisabled = "0 0 0 2px #f4f4f4, 0 0 0 5px #DADBDC";
export const BoxshadowButtonPressDefault = "0 0 0 2px #ffffff, 0 0 0 5px #001644";
export const BoxshadowButtonPressAlternative = "0 0 0 2px #ffffff, 0 0 0 5px #1456A1";
export const BoxshadowButtonPressInverse = "0 0 0 2px #333333, 0 0 0 5px #DADBDC";
export const BoxshadowButtonGhostDefault = "0 0 0 1px #002677";
export const BoxshadowButtonGhostAlternative = "0 0 0 1px #196ECF";
export const BoxshadowButtonGhostInverse = "0 0 0 1px #ffffff";
export const BoxshadowButtonDisabled = "0 0 0 1px #DADBDC";
export const BoxshadowButtonGroupedDefaultStatic = "0 0 0 5px #ffffff";
export const BoxshadowButtonGroupedDefaultFocus = "inset 0 0 0 3px #002677, inset 0 0 0 6px #ffffff";
export const BoxshadowButtonGroupedDefaultSelected = "0 0 0 3px #001E5E";
export const BoxshadowLinkBase = "0 0 0 3px #196ECF";
export const BoxshadowLinkInverse = "0 0 0 3px #E5F8FB";
export const BoxshadowLinkVisited = "0 0 0 3px #E5F8FB";
export const BoxshadowFocus = "0 0 0 2px #ffffff, 0 0 0 3px #196ECF";
export const BoxshadowTableFocusHeader = "0 0 0 1px #ffffff, inset 0 0 0 2px #196ECF ";
export const BoxshadowTableFocusRow = "0 0 0 1px #ffffff, inset 0 0 0 2px #196ECF ";
export const BoxshadowTableExpandButtonDefault = "0 0 0 1px #5a5a5a";
export const BoxshadowTableExpandButtonFocus = "0 0 0 2px #fafafa, 0 0 0 5px #196ECF";
export const BoxshadowTableExpandButtonPress = "0 0 0 2px #fafafa, 0 0 0 5px #196ECF";
export const BoxshadowNavigationGlobalButtonFocus = "0 0 0 3px #002677";
export const BoxshadowNavigationGlobalButtonPress = "0 0 0 2px #ffffff, 0 0 0 5px #001644";
export const BoxshadowNavigationGlobalButtonStaticFocus = "0 0 0 2px #ffffff, 0 0 0 5px #196ECF";
export const BoxshadowNavigationGlobalButtonStaticPress = "0 0 0 2px #ffffff, 0 0 0 5px #196ECF";
export const BoxshadowTabUnselected = "0 4px 0 0 #F4F4F4";
export const BoxshadowTabSelected = "0 4px 0 0 #002677";
export const BoxshadowPanelsBase = "0 0 0 1px #DADBDC";
export const BoxshadowPanelsAccessible = "0 0 0 1px #DADBDC";
export const BoxshadowPanelsError = "0 0 0 1px #E91B18";
export const ColorBackgroundBase = "#ffffff";
export const ColorBackgroundBrandBase = "#002677";
export const ColorBackgroundBrandHighlight = "#002677";
export const ColorBackgroundBrandAccent = "#E5F8FB";
export const ColorBackgroundPrimaryBase = "#002677";
export const ColorBackgroundPrimaryHighlight = "#196ECF";
export const ColorBackgroundPrimaryAccent = "#E5F8FB";
export const ColorBackgroundSecondaryBase = "#196ECF";
export const ColorBackgroundSecondaryHighlight = "#196ECF";
export const ColorBackgroundSecondaryAccent = "#E5F8FB";
export const ColorBackgroundNeutralBase = "#333333";
export const ColorBackgroundNeutralHighlight = "#6f6f6f";
export const ColorBackgroundNeutralAccent = "#F4F4F4";
export const ColorBackgroundInverse = "#333333";
export const ColorBackgroundOverlay = "#6f6f6f";
export const ColorBackgroundAccentBase = "#6f6f6f";
export const ColorBackgroundAccentSubtle = "#f4f4f4";
export const ColorBackgroundAccentStrong = "#5a5a5a";
export const ColorBackgroundSurfaceBase = "#ffffff";
export const ColorBackgroundSurfaceAccentStrong = "#002677";
export const ColorBackgroundSurfaceAccentSubtlePrimary = "#E5F8FB";
export const ColorBackgroundSurfaceAccentSubtleNeutral = "#f4f4f4";
export const ColorBackgroundSurfaceHover = "#002677";
export const ColorBackgroundSurfaceInverseBase = "#333333";
export const ColorBackgroundSurfaceInverseAccentStrong = "#F4F4F4";
export const ColorBackgroundSurfaceInverseAccentSubtle = "#6f6f6f";
export const ColorBackgroundFormElementBase = "#ffffff";
export const ColorBackgroundFormElementHover = "#002677";
export const ColorBackgroundFormElementFocus = "#196ECF";
export const ColorBackgroundFormElementError = "#E91B18";
export const ColorBackgroundTableHeaderBase = "#ffffff";
export const ColorBackgroundTableHeaderAccent = "#E5F8FB";
export const ColorBackgroundTableRowBase = "#ffffff";
export const ColorBackgroundTableRowAccentStrong = "#6f6f6f";
export const ColorBackgroundTableRowAccentSubtle = "#fafafa";
export const ColorBackgroundTableRowAccentSlight = "#f4f4f4";
export const ColorBackgroundTableRowInteractionStrong = "#333333";
export const ColorBackgroundTableRowInteractionSubtle = "#F4F4F4";
export const ColorBackgroundTableNestedHeaderBase = "#333333";
export const ColorBackgroundTableNestedHeaderAccent = "#E5F8FB";
export const ColorBackgroundTableNestedRowBase = "#fafafa";
export const ColorBackgroundTableNestedRowAccent = "#F4F4F4";
export const ColorBackgroundTableNestedInteractionStrong = "#002677";
export const ColorBackgroundTableNestedInteractionSubtle = "#E5F8FB";
export const ColorBackgroundNavigationGlobalBase = "#ffffff";
export const ColorBackgroundNavigationGlobalAccentBase = "#196ECF";
export const ColorBackgroundNavigationGlobalAccentStrong = "#001E5E";
export const ColorBackgroundNavigationGlobalAccentStronger = "#001644";
export const ColorBackgroundNavigationGlobalSurfaceBase = "#333333";
export const ColorBackgroundNavigationGlobalSurfaceSelected = "#E5F8FB";
export const ColorBackgroundNavigationGlobalSurfaceHover = "#002677";
export const ColorBackgroundNavigationPrimaryBase = "#ffffff";
export const ColorBackgroundNavigationPrimaryAccent = "#002677";
export const ColorBackgroundNavigationPrimarySurfaceBase = "#333333";
export const ColorBackgroundNavigationPrimarySurfaceSelected = "#E5F8FB";
export const ColorBackgroundNavigationPrimarySurfaceHover = "#002677";
export const ColorBackgroundNavigationResponsiveBase = "#ffffff";
export const ColorBackgroundNavigationResponsiveAccentStrong = "#196ECF";
export const ColorBackgroundNavigationResponsiveAccentSubtle = "#F4F4F4";
export const ColorBorderBase = "#DADBDC";
export const ColorBorderAccessible = "#737373";
export const ColorBorderSubtle = "#f4f4f4";
export const ColorBorderInverseBase = "#DADBDC";
export const ColorBorderInverseAccessible = "#ffffff";
export const ColorBorderInverseSubtle = "#737373";
export const ColorBorderBrandBase = "#002677";
export const ColorBorderBrandHighlight = "#002677";
export const ColorBorderPrimaryBase = "#002677";
export const ColorBorderPrimaryHighlight = "#196ECF";
export const ColorBorderSecondaryBase = "#196ECF";
export const ColorBorderSecondaryHighlight = "#196ECF";
export const ColorBorderSurfaceFocusHover = "#ffffff";
export const ColorBorderFormElementBase = "#737373";
export const ColorBorderFormElementActive = "#196ECF";
export const ColorBorderFormElementFocus = "#196ECF";
export const ColorBorderTableHeaderBase = "#8D8D8D";
export const ColorBorderTableHeaderAccentStrong = "#333333";
export const ColorBorderTableHeaderAccentSubtle = "#F4F4F4";
export const ColorBorderTableRowBase = "#DADBDC";
export const ColorBorderTableRowAccentStrong = "#5a5a5a";
export const ColorBorderTableRowAccentSubtle = "#F4F4F4";
export const ColorBorderTableRowAccentSlight = "#f4f4f4";
export const ColorBorderNavigationGlobalBase = "#002677";
export const ColorBorderNavigationGlobalAccentStrong = "#002677";
export const ColorBorderNavigationGlobalAccentStronger = "#001644";
export const ColorBorderNavigationGlobalAccentSubtle = "#E5F8FB";
export const ColorBorderNavigationPrimaryBase = "#002677";
export const ColorBorderNavigationPrimaryAccentStrong = "#196ECF";
export const ColorBorderNavigationPrimaryAccentSubtle = "#E5F8FB";
export const ColorBorderNavigationResponsiveBase = "#DADBDC";
export const ColorBorderNavigationResponsiveAccentStrong = "#196ECF";
export const ColorBorderNavigationResponsiveAccentSubtle = "#E5F8FB";
export const ColorButtonDefaultBase = "#002677";
export const ColorButtonDefaultFocus = "#001E5E";
export const ColorButtonDefaultPress = "#001644";
export const ColorButtonAlternativeBase = "#196ECF";
export const ColorButtonAlternativeFocus = "#1662B8";
export const ColorButtonAlternativePress = "#1456A1";
export const ColorButtonInverseBase = "#ffffff";
export const ColorButtonInverseFocus = "#f4f4f4";
export const ColorButtonInversePress = "#DADBDC";
export const ColorButtonDisabled = "#DADBDC";
export const ColorLinkBase = "#196ECF";
export const ColorLinkPress = "#002677";
export const ColorLinkInverse = "#E5F8FB";
export const ColorLinkVisited = "#E5F8FB";
export const ColorTextBrand = "#002677";
export const ColorTextPrimary = "#002677";
export const ColorTextSecondary = "#196ECF";
export const ColorTextContrastBrand = "#ffffff";
export const ColorTextContrastPrimary = "#ffffff";
export const ColorTextContrastSecondary = "#ffffff";
export const ColorTextSurfaceHover = "#ffffff";
export const ColorTextBase = "#333333";
export const ColorTextInverse = "#ffffff";
export const ColorTextHeadingH1 = "#002677";
export const ColorTextHeadingH2 = "#002677";
export const ColorTextHeadingH3 = "#002677";
export const ColorTextHeadingH4 = "#196ECF";
export const ColorTextHeadingH5 = "#002677";
export const ColorTextHeadingH6 = "#002677";
export const ColorTextSubheading = "#000000";
export const ColorTextHero = "#002677";
export const ColorTextFormsLabel = "#5a5a5a";
export const ColorTextFormsHelper = "#6F6F6F";
export const ColorTextFormsError = "#ffffff";
export const ColorTextTableHeader = "#333333";
export const ColorTextTableRow = "#5a5a5a";
export const ColorTextTableNestedtableHeader = "#ffffff";
export const ColorTextNavigationGlobal = "#5a5a5a";
export const ColorTextNavigationPrimary = "#333333";
export const ColorTextNavigationResponsive = "#196ECF";
export const ColorSemanticSuccessBase = "#0D880B";
export const ColorSemanticSuccessHighlight = "#ECFDEC";
export const ColorSemanticSuccessStrong = "#0D880B";
export const ColorSemanticSuccessSubtle = "#ECFDEC";
export const ColorSemanticSuccessBorder = "#0D880B";
export const ColorSemanticSuccessContent = "#ffffff";
export const ColorSemanticSuccessContrast = "#333333";
export const ColorSemanticErrorBase = "#E91B18";
export const ColorSemanticErrorHighlight = "#F9F1ED";
export const ColorSemanticErrorStrong = "#E91B18";
export const ColorSemanticErrorSubtle = "#F9F1ED";
export const ColorSemanticErrorBorder = "#E91B18";
export const ColorSemanticErrorContent = "#ffffff";
export const ColorSemanticErrorContrast = "#333333";
export const ColorSemanticWarningBase = "#D14600";
export const ColorSemanticWarningHighlight = "#F9F1ED";
export const ColorSemanticWarningStrong = "#D14600";
export const ColorSemanticWarningSubtle = "#F9F1ED";
export const ColorSemanticWarningBorder = "#D14600";
export const ColorSemanticWarningContent = "#ffffff";
export const ColorSemanticWarningContrast = "#333333";
export const ColorSemanticInfoBase = "#196ECF";
export const ColorSemanticInfoHighlight = "#E5F8FB";
export const ColorSemanticInfoStrong = "#196ECF";
export const ColorSemanticInfoSubtle = "#E5F8FB";
export const ColorSemanticInfoBorder = "#196ECF";
export const ColorSemanticInfoContent = "#ffffff";
export const ColorSemanticInfoContrast = "#333333";
export const ColorFocus = "#196ECF";
export const ColorDisabledBackground = "#f4f4f4";
export const ColorDisabledBorder = "#DADBDC";
export const ColorDisabledText = "#8d8d8d";
export const ColorShadowBase = "51, 51, 51";
export const ColorShadowBrand = "0, 38, 119";
export const ColorShadowAccessible = "115, 115, 115";
export const ElevationLower = "0 4px 24px 0 rgba(51, 51, 51, 0.15) inset";
export const ElevationLow = "1px 0 8px 0 rgba(51, 51, 51, 0.25) inset";
export const ElevationMedium = "1px 0 8px 0 rgba(51, 51, 51, 0.25)";
export const ElevationHigh = "1px 0 4px 0 rgba(51, 51, 51, 0.25)";
export const ElevationHigherHorizontalBottom = "0 1px 4px 0 rgba(51, 51, 51, 0.15)";
export const ElevationHigherHorizontalTop = "1px 0 4px 0 rgba(51, 51, 51, 0.15)";
export const ElevationHigherVerticalLeft = "1px 0 4px 0 rgba(51, 51, 51, 0.15)";
export const ElevationHigherVerticalRight = "0 1px 4px 0 rgba(51, 51, 51, 0.15)";
export const ElevationHighest = "0px 4px 24px 0px rgba(51, 51, 51, 0.15)";
export const FontSizeHero = "5rem";
export const FontSizeHeadingH1 = "2.75rem";
export const FontSizeHeadingH2 = "2rem";
export const FontSizeHeadingH3 = "1.5rem";
export const FontSizeHeadingH4 = "1.25rem";
export const FontSizeHeadingH5 = "1.125rem";
export const FontSizeHeadingH6 = "1rem";
export const FontSizeBodyTiny = "0.75rem";
export const FontSizeBodySmall = "0.875rem";
export const FontSizeBodyBase = "1rem";
export const FontSizeBodyLarge = "1rem";
export const FontSizeBodyLead = "1.5rem";
export const FontSizeFormLabel = "1rem";
export const FontSizeFormHelp = "0.75rem";
export const FontSizeFormError = "0.75rem";
export const FontSizeTableHeader = "1rem";
export const FontSizeTableRow = "1rem";
export const FontSizeNavigationGlobal = "0.75rem";
export const FontSizeNavigationPrimary = "1rem";
export const FontWeightLight = "400";
export const FontWeightBase = "400";
export const FontWeightMedium = "600";
export const FontWeightStrong = "700";
export const FontWeightFormLabels = "700";
export const FontWeightFormHelper = "400";
export const FontWeightHeadingH1 = "600";
export const FontWeightHeadingH2 = "600";
export const FontWeightHeadingH3 = "600";
export const FontWeightHeadingH4 = "700";
export const FontWeightHeadingH5 = "700";
export const FontWeightHeadingH6 = "700";
export const FontFamilyHeadingH1 = "UHC Serif Headline Semibold";
export const FontFamilyHeadingH2 = "UHC Serif Headline Semibold";
export const FontFamilyHeadingH3 = "UHC Serif Headline Semibold";
export const FontFamilyHeadingH4 = "UHC Sans";
export const FontFamilyHeadingH5 = "UHC Sans";
export const FontFamilyHeadingH6 = "UHC Sans";
export const FontFamilyBrand = "UHC Serif";
export const FontFamilyBase = "UHC Sans";
export const FontFamilyData = "UHC Sans";
export const FontFamilyMonotype = "UHC Sans";
export const FontFamilySecondary = "UHC Sans";
export const FontFamilyLabels = "UHC Sans";
export const FontLetterspacingBase = "0.3rem";
export const FontLetterspacingLoose = "1rem";
export const FontLetterspacingTight = "-0.3rem";
export const FontLineheightBase = "1.5rem";
export const FontLineheightXs = "1rem";
export const FontLineheightS = "1.125rem";
export const FontLineheightM = "1.25rem";
export const FontLineheightL = "1.75rem";
export const FontLineheightXl = "2rem";
export const LogoWidthFooter = "10rem";
export const OpacityLowest = "0.15";
export const OpacityLow = "0.25";
export const OpacityMedium = "0.30";
export const OpacityHigh = "0.50";
export const OpacityHighest = "0.70";
export const SpacingXxs = "0.25rem";
export const SpacingXs = "0.5rem";
export const SpacingSm = "0.75rem";
export const SpacingBase = "1rem";
export const SpacingM = "1.5rem";
export const SpacingLg = "2rem";
export const SpacingXl = "2.5rem";
export const SpacingXxl = "3.5rem";
