// @uitk/themes/optum/js/_variables.js
/**
 * Do not edit directly
 * Generated on Wed, 27 Jul 2022 10:16:51 GMT
 */

export const AliasBorderWidthBase = "1px";
export const AliasBorderWidthMedium = "2px";
export const AliasBorderWidthLarge = "4px";
export const AliasBorderWidthPanels = "2px";
export const AliasBorderWidthButtons = "2px";
export const AliasBorderWidthFormElementBase = "1px";
export const AliasBorderWidthFormElementFocus = "3px";
export const AliasBorderWidthFocus = "5px";
export const AliasBorderRadiusButtons = "1.5rem";
export const AliasBorderRadiusBase = "0.25rem";
export const AliasBorderRadiusFormElement = "0.25rem";
export const AliasBorderRadiusSmall = "0.125rem";
export const AliasBorderRadiusLarge = "0.5rem";
export const AliasBoxshadowFormBase = "0 0 0 1px #909090";
export const AliasBoxshadowFormHover = "0 0 0 1px #ff612b";
export const AliasBoxshadowFormFocus = "0 0 0 3px #ff612b";
export const AliasBoxshadowFormError = "0 0 0 1px #C40000";
export const AliasBoxshadowFormDisabled = "0 0 0 1px #cccccc";
export const AliasBoxshadowFormCheckedFocus = "0 0 0 2px #ffffff, 0 0 0 5px #ff612b";
export const AliasBoxshadowFormCheckedErrorFocus = "0 0 0 2px #ffffff, 0 0 0 5px #C40000";
export const AliasBoxshadowFormInputFocusError = "0 0 0 3px #C40000";
export const AliasBoxshadowButtonFocusDefault = "0 0 0 2px #ffffff, 0 0 0 5px #001e5f";
export const AliasBoxshadowButtonFocusAlternative = "0 0 0 2px #ffffff, 0 0 0 5px #333333";
export const AliasBoxshadowButtonFocusInverse = "0 0 0 2px #222222, 0 0 0 5px #fbfbfb";
export const AliasBoxshadowButtonFocusDisabled = "0 0 0 2px #fbfbfb, 0 0 0 5px #cccccc";
export const AliasBoxshadowButtonPressDefault = "0 0 0 2px #ffffff, 0 0 0 5px #001747";
export const AliasBoxshadowButtonPressAlternative = "0 0 0 2px #ffffff, 0 0 0 5px #111111";
export const AliasBoxshadowButtonPressInverse = "0 0 0 2px #222222, 0 0 0 5px #cccccc";
export const AliasBoxshadowButtonGhostDefault = "0 0 0 2px #002677";
export const AliasBoxshadowButtonGhostAlternative = "0 0 0 2px #444444";
export const AliasBoxshadowButtonGhostInverse = "0 0 0 2px #ffffff";
export const AliasBoxshadowButtonDisabled = "0 0 0 2px #cccccc";
export const AliasBoxshadowButtonGroupedDefaultStatic = "inset 0 0 0 2px #002677";
export const AliasBoxshadowButtonGroupedDefaultFocus = "inset 0 0 0 3px #002677, inset 0 0 0 6px #ffffff";
export const AliasBoxshadowButtonGroupedDefaultSelected = "0 0 0 3px #002677";
export const AliasBoxshadowLinkBase = "0 0 0 3px #3271e1";
export const AliasBoxshadowLinkInverse = "0 0 0 3px #d9f6fa";
export const AliasBoxshadowLinkVisited = "0 0 0 3px #2B60BF";
export const AliasBoxshadowFocus = "0 0 0 3px #ff612b";
export const AliasBoxshadowTableFocusHeader = "0 0 0 1px #ffffff, inset 0 0 0 2px #002677";
export const AliasBoxshadowTableFocusRow = "0 0 0 1px #ffffff, inset 0 0 0 2px #002677";
export const AliasBoxshadowTableExpandButtonDefault = "0 0 0 1px #333333";
export const AliasBoxshadowTableExpandButtonFocus = "0 0 0 2px #f5f5f5, 0 0 0 5px #333333";
export const AliasBoxshadowTableExpandButtonPress = "0 0 0 2px #f5f5f5, 0 0 0 5px #333333";
export const AliasBoxshadowNavigationGlobalButtonFocus = " 0 0 0 3px #ff612b";
export const AliasBoxshadowNavigationGlobalButtonPress = "0 0 0 2px #ffffff, 0 0 0 5px #001747";
export const AliasBoxshadowNavigationGlobalButtonStaticFocus = "0 0 0 2px #ffffff, 0 0 0 5px #ff612b";
export const AliasBoxshadowNavigationGlobalButtonStaticPress = "0 0 0 2px #ffffff, 0 0 0 5px #ff612b";
export const AliasBoxshadowTabUnselected = "0 4px 0 0 #fbfbfb";
export const AliasBoxshadowTabSelected = "0 4px 0 0 #ff612b";
export const AliasBoxshadowPanelsBase = "0 0 0 2px #cccccc";
export const AliasBoxshadowPanelsAccessible = "0 0 0 2px #909090";
export const AliasBoxshadowPanelsError = "0 0 0 2px #C40000";
export const AliasPaletteBrand10 = "#fbf9f4";
export const AliasPaletteBrand70 = "#ff612b";
export const AliasPaletteBrand100 = "#ff612b";
export const AliasPaletteBrand110 = "#cc4e22";
export const AliasPaletteBrand120 = "#993a1a";
export const AliasPalettePrimary10 = "#d9f6fa";
export const AliasPalettePrimary70 = "#3967d8";
export const AliasPalettePrimary100 = "#002677";
export const AliasPalettePrimary110 = "#001e5f";
export const AliasPalettePrimary120 = "#001747";
export const AliasPaletteSecondary10 = "#f5f5f5";
export const AliasPaletteSecondary70 = "#959595";
export const AliasPaletteSecondary100 = "#444444";
export const AliasPaletteSecondary110 = "#333333";
export const AliasPaletteSecondary120 = "#111111";
export const AliasPaletteNeutral0 = "#ffffff";
export const AliasPaletteNeutral10 = "#fbfbfb";
export const AliasPaletteNeutral20 = "#f5f5f5";
export const AliasPaletteNeutral30 = "#e0e0e0";
export const AliasPaletteNeutral40 = "#cccccc";
export const AliasPaletteNeutral50 = "#b3b3b3";
export const AliasPaletteNeutral60 = "#909090";
export const AliasPaletteNeutral70 = "#8e8e8e";
export const AliasPaletteNeutral80 = "#5a5a5a";
export const AliasPaletteNeutral90 = "#333333";
export const AliasPaletteNeutral100 = "#222222";
export const AliasPaletteSuccess10 = "#E5F1E5";
export const AliasPaletteSuccess70 = "#99C699";
export const AliasPaletteSuccess100 = "#007000";
export const AliasPaletteSuccess120 = "#004300";
export const AliasPaletteSuccessBorder = "#007000";
export const AliasPaletteSuccessContent = "#ffffff";
export const AliasPaletteSuccessContrast = "#111111";
export const AliasPaletteError10 = "#F9E5E5";
export const AliasPaletteError70 = "#E79999";
export const AliasPaletteError100 = "#C40000";
export const AliasPaletteError120 = "#760000";
export const AliasPaletteErrorBorder = "#C40000";
export const AliasPaletteErrorContent = "#ffffff";
export const AliasPaletteErrorContrast = "#111111";
export const AliasPaletteInfo10 = "#E5F2F3";
export const AliasPaletteInfo70 = "#99CBD0";
export const AliasPaletteInfo100 = "#007c89";
export const AliasPaletteInfo120 = "#004952";
export const AliasPaletteInfoBorder = "#004A52";
export const AliasPaletteInfoContent = "#ffffff";
export const AliasPaletteInfoContrast = "#111111";
export const AliasPaletteWarning10 = "#FEF8E5";
export const AliasPaletteWarning70 = "#FBE299";
export const AliasPaletteWarning100 = "#F5B700";
export const AliasPaletteWarning120 = "#936E00";
export const AliasPaletteWarningBorder = "#936E00";
export const AliasPaletteWarningContent = "#111111";
export const AliasPaletteWarningContrast = "#ffffff";
export const AliasPaletteLink10 = "#d9f6fa";
export const AliasPaletteLink70 = "#2B60BF";
export const AliasPaletteLink100 = "#3271e1";
export const AliasPaletteLink120 = "#002677";
export const AliasPaletteLight = "#ffffff";
export const AliasPaletteDark = "#111111";
export const AliasPaletteFocus = "#ff612b";
export const AliasPaletteActive = "#ff612b";
export const AliasPaletteSurfaceHover = "#D9F6FA";
export const AliasPaletteSurfaceTextHover = "#002677";
export const AliasPaletteSurfaceBorderHover = "#002677";
export const AliasPaletteDisabledBackground = "#fbfbfb";
export const AliasPaletteDisabledBorder = "#cccccc";
export const AliasPaletteDisabledText = "#b3b3b3";
export const AliasPaletteShadowBase = "34, 34, 34";
export const AliasPaletteShadowBrand = "255, 97, 43";
export const AliasPaletteShadowAccessible = "142,142,142";
export const AliasFontSizeHero = "5rem";
export const AliasFontSizeHeadingH1 = "2.5rem";
export const AliasFontSizeHeadingH2 = "2rem";
export const AliasFontSizeHeadingH3 = "1.5rem";
export const AliasFontSizeHeadingH4 = "1.25rem";
export const AliasFontSizeHeadingH5 = "1.125rem";
export const AliasFontSizeHeadingH6 = "1rem";
export const AliasFontSizeBodyTiny = "0.75rem";
export const AliasFontSizeBodySmall = "0.875rem";
export const AliasFontSizeBodyBase = "1rem";
export const AliasFontSizeBodyLarge = "1.125rem";
export const AliasFontSizeBodyLead = "1.5rem";
export const AliasFontSizeFormLabel = "1rem";
export const AliasFontSizeFormHelp = "0.75rem";
export const AliasFontSizeFormError = "0.75rem";
export const AliasFontSizeTableHeader = "1rem";
export const AliasFontSizeTableRow = "1rem";
export const AliasFontSizeNavigationGlobal = "0.75rem";
export const AliasFontSizeNavigationPrimary = "0.875rem";
export const AliasFontWeightLight = "400";
export const AliasFontWeightRegular = "400";
export const AliasFontWeightMedium = "700";
export const AliasFontWeightBold = "700";
export const AliasFontWeightHeadingH1 = "700";
export const AliasFontWeightHeadingH2 = "700";
export const AliasFontWeightHeadingH3 = "700";
export const AliasFontWeightHeadingH4 = "700";
export const AliasFontWeightHeadingH5 = "700";
export const AliasFontWeightHeadingH6 = "700";
export const AliasFontWeightFormLabels = "700";
export const AliasFontWeightFormHelper = "600";
export const AliasFontWeightFormError = "600";
export const AliasFontFamilyHeadingH1 = "Optum Sans";
export const AliasFontFamilyHeadingH2 = "Optum Sans";
export const AliasFontFamilyHeadingH3 = "Optum Sans";
export const AliasFontFamilyHeadingH4 = "Optum Sans";
export const AliasFontFamilyHeadingH5 = "Optum Sans";
export const AliasFontFamilyHeadingH6 = "Optum Sans";
export const AliasFontFamilyBrand = "Optum Sans";
export const AliasFontFamilyBase = "Optum Sans";
export const AliasFontFamilyData = "Optum Sans";
export const AliasFontFamilyMonotype = "Optum Sans";
export const AliasFontFamilySecondary = "Optum Sans";
export const AliasFontFamilyLabels = "Optum Sans";
export const AliasFontColorHeadingH1 = "#002677";
export const AliasFontColorHeadingH2 = "#002677";
export const AliasFontColorHeadingH3 = "#002677";
export const AliasFontColorHeadingH4 = "#002677";
export const AliasFontColorHeadingH5 = "#002677";
export const AliasFontColorHeadingH6 = "#002677";
export const AliasFontColorSubheading = "#002677";
export const AliasFontColorHero = "#002677";
export const AliasFontColorBase = "#333333";
export const AliasFontColorInverse = "#ffffff";
export const AliasFontColorSurfaceHover = "#002677";
export const AliasFontLetterspacingBase = "0.3rem";
export const AliasFontLetterspacingLoose = "1rem";
export const AliasFontLetterspacingTight = "-0.3rem";
export const AliasFontLineheightBase = "1.5rem";
export const AliasFontLineheightXs = "1rem";
export const AliasFontLineheightS = "1.125rem";
export const AliasFontLineheightM = "1.25rem";
export const AliasFontLineheightL = "1.75rem";
export const AliasFontLineheightXl = "2rem";
export const AliasLogoWidthFooter = "6.5rem";
export const AliasOpacityLowest = "0.15";
export const AliasOpacityLow = "0.25";
export const AliasOpacityMedium = "0.30";
export const AliasOpacityHigh = "0.50";
export const AliasOpacityHighest = "0.70";
export const AliasSpacingXxs = "0.25rem";
export const AliasSpacingXs = "0.5rem";
export const AliasSpacingSm = "0.75rem";
export const AliasSpacingBase = "1rem";
export const AliasSpacingM = "1.5rem";
export const AliasSpacingLg = "2rem";
export const AliasSpacingXl = "2.5rem";
export const AliasSpacingXxl = "3.5rem";
export const AnimationDurationAnimateSlow = "400ms";
export const AnimationDurationAnimateMedium = "200ms";
export const AnimationDurationAnimateFast = "70ms";
export const BorderWidthBase = "1px";
export const BorderWidthMedium = "2px";
export const BorderWidthLarge = "4px";
export const BorderWidthPanels = "2px";
export const BorderWidthButtons = "2px";
export const BorderWidthFormElementBase = "1px";
export const BorderWidthFormElementFocus = "3px";
export const BorderWidthFocus = "5px";
export const BorderRadiusButtons = "1.5rem";
export const BorderRadiusBase = "0.25rem";
export const BorderRadiusFormElement = "0.25rem";
export const BorderRadiusSmall = "0.125rem";
export const BorderRadiusLarge = "0.5rem";
export const BoxshadowFormBase = "0 0 0 1px #909090";
export const BoxshadowFormHover = "0 0 0 1px #ff612b";
export const BoxshadowFormFocus = "0 0 0 3px #ff612b";
export const BoxshadowFormError = "0 0 0 1px #C40000";
export const BoxshadowFormDisabled = "0 0 0 1px #cccccc";
export const BoxshadowFormCheckedFocus = "0 0 0 2px #ffffff, 0 0 0 5px #ff612b";
export const BoxshadowFormCheckedErrorFocus = "0 0 0 2px #ffffff, 0 0 0 5px #C40000";
export const BoxshadowFormInputFocusError = "0 0 0 3px #C40000";
export const BoxshadowButtonFocusDefault = "0 0 0 2px #ffffff, 0 0 0 5px #001e5f";
export const BoxshadowButtonFocusAlternative = "0 0 0 2px #ffffff, 0 0 0 5px #333333";
export const BoxshadowButtonFocusInverse = "0 0 0 2px #222222, 0 0 0 5px #fbfbfb";
export const BoxshadowButtonFocusDisabled = "0 0 0 2px #fbfbfb, 0 0 0 5px #cccccc";
export const BoxshadowButtonPressDefault = "0 0 0 2px #ffffff, 0 0 0 5px #001747";
export const BoxshadowButtonPressAlternative = "0 0 0 2px #ffffff, 0 0 0 5px #111111";
export const BoxshadowButtonPressInverse = "0 0 0 2px #222222, 0 0 0 5px #cccccc";
export const BoxshadowButtonGhostDefault = "0 0 0 2px #002677";
export const BoxshadowButtonGhostAlternative = "0 0 0 2px #444444";
export const BoxshadowButtonGhostInverse = "0 0 0 2px #ffffff";
export const BoxshadowButtonDisabled = "0 0 0 2px #cccccc";
export const BoxshadowButtonGroupedDefaultStatic = "inset 0 0 0 2px #002677";
export const BoxshadowButtonGroupedDefaultFocus = "inset 0 0 0 3px #002677, inset 0 0 0 6px #ffffff";
export const BoxshadowButtonGroupedDefaultSelected = "0 0 0 3px #002677";
export const BoxshadowLinkBase = "0 0 0 3px #3271e1";
export const BoxshadowLinkInverse = "0 0 0 3px #d9f6fa";
export const BoxshadowLinkVisited = "0 0 0 3px #2B60BF";
export const BoxshadowFocus = "0 0 0 3px #ff612b";
export const BoxshadowTableFocusHeader = "0 0 0 1px #ffffff, inset 0 0 0 2px #002677";
export const BoxshadowTableFocusRow = "0 0 0 1px #ffffff, inset 0 0 0 2px #002677";
export const BoxshadowTableExpandButtonDefault = "0 0 0 1px #333333";
export const BoxshadowTableExpandButtonFocus = "0 0 0 2px #f5f5f5, 0 0 0 5px #333333";
export const BoxshadowTableExpandButtonPress = "0 0 0 2px #f5f5f5, 0 0 0 5px #333333";
export const BoxshadowNavigationGlobalButtonFocus = " 0 0 0 3px #ff612b";
export const BoxshadowNavigationGlobalButtonPress = "0 0 0 2px #ffffff, 0 0 0 5px #001747";
export const BoxshadowNavigationGlobalButtonStaticFocus = "0 0 0 2px #ffffff, 0 0 0 5px #ff612b";
export const BoxshadowNavigationGlobalButtonStaticPress = "0 0 0 2px #ffffff, 0 0 0 5px #ff612b";
export const BoxshadowTabUnselected = "0 4px 0 0 #fbfbfb";
export const BoxshadowTabSelected = "0 4px 0 0 #ff612b";
export const BoxshadowPanelsBase = "0 0 0 2px #cccccc";
export const BoxshadowPanelsAccessible = "0 0 0 2px #cccccc";
export const BoxshadowPanelsError = "0 0 0 2px #C40000";
export const ColorBackgroundBase = "#ffffff";
export const ColorBackgroundBrandBase = "#ff612b";
export const ColorBackgroundBrandHighlight = "#ff612b";
export const ColorBackgroundBrandAccent = "#fbf9f4";
export const ColorBackgroundPrimaryBase = "#002677";
export const ColorBackgroundPrimaryHighlight = "#3967d8";
export const ColorBackgroundPrimaryAccent = "#d9f6fa";
export const ColorBackgroundSecondaryBase = "#444444";
export const ColorBackgroundSecondaryHighlight = "#959595";
export const ColorBackgroundSecondaryAccent = "#f5f5f5";
export const ColorBackgroundNeutralBase = "#222222";
export const ColorBackgroundNeutralHighlight = "#8e8e8e";
export const ColorBackgroundNeutralAccent = "#e0e0e0";
export const ColorBackgroundInverse = "#222222";
export const ColorBackgroundOverlay = "#8e8e8e";
export const ColorBackgroundAccentBase = "#8e8e8e";
export const ColorBackgroundAccentSubtle = "#fbfbfb";
export const ColorBackgroundAccentStrong = "#333333";
export const ColorBackgroundSurfaceBase = "#ffffff";
export const ColorBackgroundSurfaceAccentStrong = "#002677";
export const ColorBackgroundSurfaceAccentSubtlePrimary = "#d9f6fa";
export const ColorBackgroundSurfaceAccentSubtleNeutral = "#fbfbfb";
export const ColorBackgroundSurfaceHover = "#D9F6FA";
export const ColorBackgroundSurfaceInverseBase = "#222222";
export const ColorBackgroundSurfaceInverseAccentStrong = "#e0e0e0";
export const ColorBackgroundSurfaceInverseAccentSubtle = "#8e8e8e";
export const ColorBackgroundFormElementBase = "#ffffff";
export const ColorBackgroundFormElementHover = "#002677";
export const ColorBackgroundFormElementFocus = "#ff612b";
export const ColorBackgroundFormElementError = "#C40000";
export const ColorBackgroundTableHeaderBase = "#ffffff";
export const ColorBackgroundTableHeaderAccent = "#d9f6fa";
export const ColorBackgroundTableRowBase = "#ffffff";
export const ColorBackgroundTableRowAccentStrong = "#8e8e8e";
export const ColorBackgroundTableRowAccentSubtle = "#f5f5f5";
export const ColorBackgroundTableRowAccentSlight = "#fbfbfb";
export const ColorBackgroundTableRowInteractionStrong = "#222222";
export const ColorBackgroundTableRowInteractionSubtle = "#e0e0e0";
export const ColorBackgroundTableNestedHeaderBase = "#222222";
export const ColorBackgroundTableNestedHeaderAccent = "#d9f6fa";
export const ColorBackgroundTableNestedRowBase = "#f5f5f5";
export const ColorBackgroundTableNestedRowAccent = "#e0e0e0";
export const ColorBackgroundTableNestedInteractionStrong = "#002677";
export const ColorBackgroundTableNestedInteractionSubtle = "#d9f6fa";
export const ColorBackgroundNavigationGlobalBase = "#ffffff";
export const ColorBackgroundNavigationGlobalAccentBase = "#3967d8";
export const ColorBackgroundNavigationGlobalAccentStrong = "#001e5f";
export const ColorBackgroundNavigationGlobalAccentStronger = "#001747";
export const ColorBackgroundNavigationGlobalSurfaceBase = "#222222";
export const ColorBackgroundNavigationGlobalSurfaceSelected = "#fbf9f4";
export const ColorBackgroundNavigationGlobalSurfaceHover = "#002677";
export const ColorBackgroundNavigationPrimaryBase = "#ffffff";
export const ColorBackgroundNavigationPrimaryAccent = "#002677";
export const ColorBackgroundNavigationPrimarySurfaceBase = "#222222";
export const ColorBackgroundNavigationPrimarySurfaceSelected = "#fbf9f4";
export const ColorBackgroundNavigationPrimarySurfaceHover = "#002677";
export const ColorBackgroundNavigationResponsiveBase = "#ffffff";
export const ColorBackgroundNavigationResponsiveAccentStrong = "#444444";
export const ColorBackgroundNavigationResponsiveAccentSubtle = "#e0e0e0";
export const ColorBorderBase = "#cccccc";
export const ColorBorderAccessible = "#909090";
export const ColorBorderSubtle = "#fbfbfb";
export const ColorBorderInverseBase = "#cccccc";
export const ColorBorderInverseAccessible = "#ffffff";
export const ColorBorderInverseSubtle = "#909090";
export const ColorBorderBrandBase = "#ff612b";
export const ColorBorderBrandHighlight = "#ff612b";
export const ColorBorderPrimaryBase = "#002677";
export const ColorBorderPrimaryHighlight = "#3967d8";
export const ColorBorderSecondaryBase = "#444444";
export const ColorBorderSecondaryHighlight = "#959595";
export const ColorBorderSurfaceFocusHover = "#002677";
export const ColorBorderFormElementBase = "#909090";
export const ColorBorderFormElementActive = "#ff612b";
export const ColorBorderFormElementFocus = "#ff612b";
export const ColorBorderTableHeaderBase = "#b3b3b3";
export const ColorBorderTableHeaderAccentStrong = "#222222";
export const ColorBorderTableHeaderAccentSubtle = "#e0e0e0";
export const ColorBorderTableRowBase = "#cccccc";
export const ColorBorderTableRowAccentStrong = "#333333";
export const ColorBorderTableRowAccentSubtle = "#e0e0e0";
export const ColorBorderTableRowAccentSlight = "#fbfbfb";
export const ColorBorderNavigationGlobalBase = "#002677";
export const ColorBorderNavigationGlobalAccentStrong = "#ff612b";
export const ColorBorderNavigationGlobalAccentStronger = "#001747";
export const ColorBorderNavigationGlobalAccentSubtle = "#d9f6fa";
export const ColorBorderNavigationPrimaryBase = "#002677";
export const ColorBorderNavigationPrimaryAccentStrong = "#3967d8";
export const ColorBorderNavigationPrimaryAccentSubtle = "#d9f6fa";
export const ColorBorderNavigationResponsiveBase = "#cccccc";
export const ColorBorderNavigationResponsiveAccentStrong = "#3967d8";
export const ColorBorderNavigationResponsiveAccentSubtle = "#d9f6fa";
export const ColorButtonDefaultBase = "#002677";
export const ColorButtonDefaultFocus = "#001e5f";
export const ColorButtonDefaultPress = "#001747";
export const ColorButtonAlternativeBase = "#444444";
export const ColorButtonAlternativeFocus = "#333333";
export const ColorButtonAlternativePress = "#111111";
export const ColorButtonInverseBase = "#ffffff";
export const ColorButtonInverseFocus = "#fbfbfb";
export const ColorButtonInversePress = "#cccccc";
export const ColorButtonDisabled = "#cccccc";
export const ColorLinkBase = "#3271e1";
export const ColorLinkPress = "#002677";
export const ColorLinkInverse = "#d9f6fa";
export const ColorLinkVisited = "#2B60BF";
export const ColorTextBrand = "#ff612b";
export const ColorTextPrimary = "#002677";
export const ColorTextSecondary = "#444444";
export const ColorTextContrastBrand = "#ffffff";
export const ColorTextContrastPrimary = "#ffffff";
export const ColorTextContrastSecondary = "#ffffff";
export const ColorTextSurfaceHover = "#002677";
export const ColorTextBase = "#333333";
export const ColorTextInverse = "#ffffff";
export const ColorTextHeadingH1 = "#002677";
export const ColorTextHeadingH2 = "#002677";
export const ColorTextHeadingH3 = "#002677";
export const ColorTextHeadingH4 = "#002677";
export const ColorTextHeadingH5 = "#002677";
export const ColorTextHeadingH6 = "#002677";
export const ColorTextSubheading = "#000000";
export const ColorTextHero = "#002677";
export const ColorTextFormsLabel = "#333333";
export const ColorTextFormsHelper = "#5a5a5a";
export const ColorTextFormsError = "#ffffff";
export const ColorTextTableHeader = "#222222";
export const ColorTextTableRow = "#333333";
export const ColorTextTableNestedtableHeader = "#ffffff";
export const ColorTextNavigationGlobal = "#333333";
export const ColorTextNavigationPrimary = "#222222";
export const ColorTextNavigationResponsive = "#444444";
export const ColorSemanticSuccessBase = "#007000";
export const ColorSemanticSuccessHighlight = "#99C699";
export const ColorSemanticSuccessStrong = "#004300";
export const ColorSemanticSuccessSubtle = "#E5F1E5";
export const ColorSemanticSuccessBorder = "#007000";
export const ColorSemanticSuccessContent = "#ffffff";
export const ColorSemanticSuccessContrast = "#111111";
export const ColorSemanticErrorBase = "#C40000";
export const ColorSemanticErrorHighlight = "#E79999";
export const ColorSemanticErrorStrong = "#760000";
export const ColorSemanticErrorSubtle = "#F9E5E5";
export const ColorSemanticErrorBorder = "#C40000";
export const ColorSemanticErrorContent = "#ffffff";
export const ColorSemanticErrorContrast = "#111111";
export const ColorSemanticWarningBase = "#F5B700";
export const ColorSemanticWarningHighlight = "#FBE299";
export const ColorSemanticWarningStrong = "#936E00";
export const ColorSemanticWarningSubtle = "#FEF8E5";
export const ColorSemanticWarningBorder = "#936E00";
export const ColorSemanticWarningContent = "#111111";
export const ColorSemanticWarningContrast = "#ffffff";
export const ColorSemanticInfoBase = "#007c89";
export const ColorSemanticInfoHighlight = "#99CBD0";
export const ColorSemanticInfoStrong = "#004952";
export const ColorSemanticInfoSubtle = "#E5F2F3";
export const ColorSemanticInfoBorder = "#004A52";
export const ColorSemanticInfoContent = "#ffffff";
export const ColorSemanticInfoContrast = "#111111";
export const ColorFocus = "#ff612b";
export const ColorDisabledBackground = "#fbfbfb";
export const ColorDisabledBorder = "#cccccc";
export const ColorDisabledText = "#b3b3b3";
export const ColorShadowBase = "34, 34, 34";
export const ColorShadowBrand = "255, 97, 43";
export const ColorShadowAccessible = "142,142,142";
export const ElevationLower = "0 4px 24px 0 rgba(34, 34, 34, 0.15) inset";
export const ElevationLow = "1px 0 8px 0 rgba(34, 34, 34, 0.25) inset";
export const ElevationMedium = "1px 0 8px 0 rgba(34, 34, 34, 0.25)";
export const ElevationHigh = "1px 0 4px 0 rgba(34, 34, 34, 0.25)";
export const ElevationHigherHorizontalBottom = "0 1px 4px 0 rgba(34, 34, 34, 0.15)";
export const ElevationHigherHorizontalTop = "1px 0 4px 0 rgba(34, 34, 34, 0.15)";
export const ElevationHigherVerticalLeft = "1px 0 4px 0 rgba(34, 34, 34, 0.15)";
export const ElevationHigherVerticalRight = "0 1px 4px 0 rgba(34, 34, 34, 0.15)";
export const ElevationHighest = "0px 4px 24px 0px rgba(34, 34, 34, 0.15)";
export const FontSizeHero = "5rem";
export const FontSizeHeadingH1 = "2.5rem";
export const FontSizeHeadingH2 = "2rem";
export const FontSizeHeadingH3 = "1.5rem";
export const FontSizeHeadingH4 = "1.25rem";
export const FontSizeHeadingH5 = "1.125rem";
export const FontSizeHeadingH6 = "1rem";
export const FontSizeBodyTiny = "0.75rem";
export const FontSizeBodySmall = "0.875rem";
export const FontSizeBodyBase = "1rem";
export const FontSizeBodyLarge = "1rem";
export const FontSizeBodyLead = "1.5rem";
export const FontSizeFormLabel = "1rem";
export const FontSizeFormHelp = "0.75rem";
export const FontSizeFormError = "0.75rem";
export const FontSizeTableHeader = "1rem";
export const FontSizeTableRow = "1rem";
export const FontSizeNavigationGlobal = "0.75rem";
export const FontSizeNavigationPrimary = "0.875rem";
export const FontWeightLight = "400";
export const FontWeightBase = "400";
export const FontWeightMedium = "700";
export const FontWeightStrong = "700";
export const FontWeightFormLabels = "700";
export const FontWeightFormHelper = "600";
export const FontWeightHeadingH1 = "700";
export const FontWeightHeadingH2 = "700";
export const FontWeightHeadingH3 = "700";
export const FontWeightHeadingH4 = "700";
export const FontWeightHeadingH5 = "700";
export const FontWeightHeadingH6 = "700";
export const FontFamilyHeadingH1 = "Optum Sans";
export const FontFamilyHeadingH2 = "Optum Sans";
export const FontFamilyHeadingH3 = "Optum Sans";
export const FontFamilyHeadingH4 = "Optum Sans";
export const FontFamilyHeadingH5 = "Optum Sans";
export const FontFamilyHeadingH6 = "Optum Sans";
export const FontFamilyBrand = "Optum Sans";
export const FontFamilyBase = "Optum Sans";
export const FontFamilyData = "Optum Sans";
export const FontFamilyMonotype = "Optum Sans";
export const FontFamilySecondary = "Optum Sans";
export const FontFamilyLabels = "Optum Sans";
export const FontLetterspacingBase = "0.3rem";
export const FontLetterspacingLoose = "1rem";
export const FontLetterspacingTight = "-0.3rem";
export const FontLineheightBase = "1.5rem";
export const FontLineheightXs = "1rem";
export const FontLineheightS = "1.125rem";
export const FontLineheightM = "1.25rem";
export const FontLineheightL = "1.75rem";
export const FontLineheightXl = "2rem";
export const LogoWidthFooter = "6.5rem";
export const OpacityLowest = "0.15";
export const OpacityLow = "0.25";
export const OpacityMedium = "0.30";
export const OpacityHigh = "0.50";
export const OpacityHighest = "0.70";
export const SpacingXxs = "0.25rem";
export const SpacingXs = "0.5rem";
export const SpacingSm = "0.75rem";
export const SpacingBase = "1rem";
export const SpacingM = "1.5rem";
export const SpacingLg = "2rem";
export const SpacingXl = "2.5rem";
export const SpacingXxl = "3.5rem";
